minecraft/networking/packets/clientbound/play/__init__.py

```diff
@@ -72,7 +74,22 @@ def get_id(context):
         {'difficulty': UnsignedByte},
         {'max_players': UnsignedByte},
         {'level_type': String},
-        {'reduced_debug_info': Boolean}])
+        {'reduced_debug_info': Boolean}
+    ])
+
+
+class ServerDifficultyPacket(Packet):
+    @staticmethod
+    def get_id(context):
+        return 0x0D if context.protocol_version >= 332 else \
+               0x0E if context.protocol_version >= 318 else \
+               0x0D if context.protocol_version >= 67 else \
```

Comment:
I guess for the old pre-release versions it wouldn't make a huge difference, but the packet id changed in [15w36d ( protocol 70) to 0x0D](https://wiki.vg/index.php?title=Pre-release_protocol&oldid=6925).
aka.
```python
0x0D if context.protocol_version >= 70 else \
```

Reply:
Good spot here. Looks like this is missing on the wiki page. Should have looked at https://joodicator.github.io/mc-dev-data/ instead ;)

Ill update the PR.